t/core/etcd-mtls.t

```diff
@@ -24,6 +24,12 @@ if ($out !~ m/function:/) {
     plan('no_plan');
 }
 
+Test::Nginx::Socket::set_http_config_filter(sub {
```

Comment:
Do we need a separate generator for this test file?

Reply:
Currently gen_snippet.lua is working fine, except for a problem at https://github.com/apache/apisix/blob/c397ede02d0b304b8b3a1d5049f6c1d8a83d3804/t/core/config.t#L306-L325, but it does not affect the test results.

Reply:
So can we remove `set_http_config_filter` from this file and use the one set in the t/APISIX.pm?